battleship.py

```diff
@@ -80,7 +99,14 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]]==1:

```

Comment:
Use the variable name in place of the number.

Reply:
Modified entire code and changed variable name